static_src/components/users_invite.jsx

```diff
@@ -16,21 +16,31 @@ import { validateString } from '../util/validators';
 const USERS_INVITE_FORM_GUID = 'users-invite-form';
 
 const propTypes = {
+  inviteDisabled: React.PropTypes.bool,
+  currentUserAccess: React.PropTypes.bool,
   error: React.PropTypes.object
 };
 const defaultProps = {};
 
+function stateSetter(props) {
```

Comment:
I don't think you need to use state setter at all, unless I'm missing something. You can access error through `this.props.error` like the other things, and don't have to set state at all.